### PR TITLE
added profile-module

### PR DESCRIPTION
**Pull request by sharma-Paras**

### CODE REVIEW

File: src/app/modules/dashboard/modules/profile/profile.component.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<div class="profile-component">
+    <div class="profile-content">
+    <div class="card-shadow-default margin-bottom profile-header ">
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/profile/profile.component.html:3]: extend utilities in scss and remove from html

File: src/app/modules/dashboard/modules/profile/profile.component.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                    </div>
+                </div>
+            </mat-card-content>
+        </mat-card>
+    </div>
+    <div class="dashboard">
+        <mat-card>
+            <mat-card-content>
+                <div class="dashboard-header">
+                    <div class="dashboard-header-content pl-8">
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/profile/profile.component.html:50]: move pl-8 to scss

File: src/app/modules/dashboard/modules/profile/profile.component.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                      </mat-grid-list>
+                </div>
+            </mat-card-content>
+        </mat-card>
+
+    </div>
+    <div class="experience">
+        <mat-card>
+            <mat-card-content>
+                <div class="experience-header pad-8">
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/profile/profile.component.html:90]: remove pad-8

File: src/app/modules/dashboard/modules/profile/profile.component.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+                    <div class="experience-header-content pl-8" style="font-size: 1.7em;">Experience</div>
+                    <div class="flex-spacer-x"></div>
+                    <div>
+                        <button mat-button class="create">
+                            <mat-icon>create</mat-icon>
+                        </button>
+                    </div>
+                </div>
+                <!-- <mat-list>
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/profile/profile.component.html:99]: remove commented code or add a comment stating the reason for keeping it.

File: src/app/modules/dashboard/modules/profile/profile.component.scss [hunk 5]
```diff
@@ -0,0 +1,97 @@
+@import "../../dashboard.component.scss";
+.profile-component {
+    display: flex;
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/profile/profile.component.scss:3]: you can use .flex-row class

Comment (sharma-Paras) [reply src/app/modules/dashboard/modules/profile/profile.component.scss]: added .flex-row class

**Pull request merged**